R/carehomes.R

```diff
@@ -1070,87 +1239,195 @@ carehomes_parameters_waning <- function(waning_rate) {
 ##'
 ##' @title Carehomes progression parameters
 ##'
-##' @return A list of parameter values
+##' @param dt The step size
 ##'
-##' @param rel_gamma_A Vector of relative rates of gamma_A for each
-##'   strain modelled. If `1` all strains have same rates. Otherwise vector of
-##'   same length as `strain_transmission`, with entries that determines the
-##'   relative scaling of the defaults for each strain.
+##' @section Time-varying parameters:
+##' Every time varying parameter has the same format, which can be `NULL` (in
+##'   which case a default single value is used) or a list with `date` and
+##'   `value` for changes in the parameter. If `value` is scalar then
+##'   `date` can be `NULL` or unspecified. If `value` is a vector then `date`
```

Comment:
Unclear to me what an 'unspecified' date, to me 'unspecified' = NULL and 'missing' = not provided.

Reply:
Have changed this to 'missing' wherever I used 'unspecified'